src/openforms/js/components/admin/form_design/NewStepFormDefinitionPicker.js

```diff
@@ -40,39 +46,61 @@ const NewStepFormDefinitionPicker = ({ onReplace }) => {
-                <FAIcon icon="recycle" extraClassname="fa-2x" title="Selecteer definitie" />
-                <span>Gebruik bestaande formulierdefinitie</span>
+                <FAIcon
+                    icon="recycle"
+                    extraClassname="fa-2x"
+                    title={intl.formatMessage({description: 'select definition icon title', defaultMessage: 'Select definition'})}
+                />
+                <span>
+                    <FormattedMessage
+                        description="Select form definition tile"
+                        defaultMessage="Select existing form definition" />
+                </span>
             </button>
 
             <span className="tiles__separator">&bull;</span>
 
             <button type="button" className="tiles__tile" onClick={() => {onReplace('')}}>
-                <FAIcon icon="pencil-square-o" extraClassname="fa-2x" title="Maak definitie" />
-                <span>Maak een nieuwe formulierdefinitie</span>
+                <FAIcon
+                    icon="pencil-square-o"
+                    extraClassname="fa-2x"
+                    title={intl.formatMessage({description: 'create form definition icon title', defaultMessage: 'Create definition'})}
+                />
+                <span>
+                    <FormattedMessage
+                        description="Create form definition tile"
+                        defaultMessage="Create a new form definition" />
+                </span>
             </button>
 
             <FormModal
                 isOpen={isModalOpen}
                 closeModal={closeModal}
-                title="Gebruik bestaande formulierdefinitie"
+                title={<FormattedMessage
+                        description="Form definition selection modal title"
+                        defaultMessage="Use existing form definition" />}
             >
                 <FormRow>
                     <Field
                         name="form-definition"
-                        label="Selecteer formulierdefinitie"
+                        label={<FormattedMessage description="Form definition select label" defaultMessage="Select form definition" />}
                         errors={validationErrors}
                         required
                     >
                         <Select
-                            name="Form definition"
                             choices={formDefinitionChoices}
                             value={selectedFormDefinition}
                             onChange={onSelectChange}
                         />
                     </Field>
                 </FormRow>
 
-                <SubmitRow onSubmit={onFormDefinitionConfirmed} btnText="Bevestigen" isDefault preventDefault />
+                <SubmitRow
+                    onSubmit={onFormDefinitionConfirmed}
+                    btnText={<FormattedMessage description="Form definition select confirm button" defaultMessage="Confirm" />}
```

Comment:
The button doesn't seem to render the translated text properly?
![Screenshot from 2021-08-12 11-34-54](https://user-images.githubusercontent.com/19154114/129174677-331fc2bc-3d07-4efd-beda-b2c9fe80fc69.png)

Reply:
d'oh, didn't test that properly :see_no_evil: